### ISSUE TITLE
failed to deploy kubernetes 

### ISSUE THREAD

**Issue by Madhu-1** [gluster/gluster-kubernetes#549]
```
TASK [master : kubeadm init] ***************************************************
fatal: [master]: FAILED! => {"changed": true, "cmd": ["kubeadm", "init", "--skip-preflight-checks", "--token=abcdef.1234567890abcdef", "--kubernetes-version=v1.13.1", "--apiserver-advertise-address=192.168.10.90"], "delta": "0:00:00.023834", "end": "2019-01-07 08:23:52.566105", "msg": "non-zero return code", "rc": 1, "start": "2019-01-07 08:23:52.542271", "stderr": "Error: unknown flag: --skip-preflight-checks\nUsage:\n  kubeadm init [flags]\n  kubeadm init [command]\n\nAvailable Commands:\n  phase       use this command to invoke single phase of the init workflow\n\nFlags:\n      --apiserver-advertise-address string   The IP address the API Server will advertise it's listening on. Specify '0.0.0.0' to use the address of the default network interface.\n      --apiserver-bind-port int32            Port for the API Server to bind to. (default 6443)\n      --apiserver-cert-extra-sans strings    Optional extra Subject Alternative Names (SANs) to use for the API Server serving certificate. Can be both IP addresses and DNS names.\n      --cert-dir string                      The path where to save and store the certificates. (default \"/etc/kubernetes/pki\")\n      --config string                        Path to kubeadm config file. WARNING: Usage of a configuration file is experimental.\n      --cri-socket string                    Specify the CRI socket to connect to. (default \"/var/run/dockershim.sock\")\n      --dry-run                              Don't apply any changes; just output what would be done.\n      --feature-gates string                 A set of key=value pairs that describe feature gates for various features. Options are:\n                                             \n  -h, --help                                 help for init\n      --ignore-preflight-errors strings      A list of checks whose errors will be shown as warnings. Example: 'IsPrivilegedUser,Swap'. Value 'all' ignores errors from all checks.\n      --image-repository string              Choose a container registry to pull control plane images from (default \"k8s.gcr.io\")\n      --kubernetes-version string            Choose a specific Kubernetes version for the control plane. (default \"stable-1\")\n      --node-name string                     Specify the node name.\n      --pod-network-cidr string              Specify range of IP addresses for the pod network. If set, the control plane will automatically allocate CIDRs for every node.\n      --service-cidr string                  Use alternative range of IP address for service VIPs. (default \"10.96.0.0/12\")\n      --service-dns-domain string            Use alternative domain for services, e.g. \"myorg.internal\". (default \"cluster.local\")\n      --skip-phases strings                  List of phases to be skipped\n      --skip-token-print                     Skip printing of the default bootstrap token generated by 'kubeadm init'.\n      --token string                         The token to use for establishing bidirectional trust between nodes and masters. The format is [a-z0-9]{6}\\.[a-z0-9]{16} - e.g. abcdef.0123456789abcdef\n      --token-ttl duration                   The duration before the token is automatically deleted (e.g. 1s, 2m, 3h). If set to '0', the token will never expire (default 24h0m0s)\n\nGlobal Flags:\n      --log-file string   If non-empty, use this log file\n      --rootfs string     [EXPERIMENTAL] The path to the 'real' host root filesystem.\n      --skip-headers      If true, avoid header prefixes in the log messages\n  -v, --v Level           log level for V logs\n\nUse \"kubeadm init [command] --help\" for more information about a command.\n\nerror: unknown flag: --skip-preflight-checks", "stderr_lines": ["Error: unknown flag: --skip-preflight-checks", "Usage:", "  kubeadm init [flags]", "  kubeadm init [command]", "", "Available Commands:", "  phase       use this command to invoke single phase of the init workflow", "", "Flags:", "      --apiserver-advertise-address string   The IP address the API Server will advertise it's listening on. Specify '0.0.0.0' to use the address of the default network interface.", "      --apiserver-bind-port int32            Port for the API Server to bind to. (default 6443)", "      --apiserver-cert-extra-sans strings    Optional extra Subject Alternative Names (SANs) to use for the API Server serving certificate. Can be both IP addresses and DNS names.", "      --cert-dir string                      The path where to save and store the certificates. (default \"/etc/kubernetes/pki\")", "      --config string                        Path to kubeadm config file. WARNING: Usage of a configuration file is experimental.", "      --cri-socket string                    Specify the CRI socket to connect to. (default \"/var/run/dockershim.sock\")", "      --dry-run                              Don't apply any changes; just output what would be done.", "      --feature-gates string                 A set of key=value pairs that describe feature gates for various features. Options are:", "                                             ", "  -h, --help                                 help for init", "      --ignore-preflight-errors strings      A list of checks whose errors will be shown as warnings. Example: 'IsPrivilegedUser,Swap'. Value 'all' ignores errors from all checks.", "      --image-repository string              Choose a container registry to pull control plane images from (default \"k8s.gcr.io\")", "      --kubernetes-version string            Choose a specific Kubernetes version for the control plane. (default \"stable-1\")", "      --node-name string                     Specify the node name.", "      --pod-network-cidr string              Specify range of IP addresses for the pod network. If set, the control plane will automatically allocate CIDRs for every node.", "      --service-cidr string                  Use alternative range of IP address for service VIPs. (default \"10.96.0.0/12\")", "      --service-dns-domain string            Use alternative domain for services, e.g. \"myorg.internal\". (default \"cluster.local\")", "      --skip-phases strings                  List of phases to be skipped", "      --skip-token-print                     Skip printing of the default bootstrap token generated by 'kubeadm init'.", "      --token string                         The token to use for establishing bidirectional trust between nodes and masters. The format is [a-z0-9]{6}\\.[a-z0-9]{16} - e.g. abcdef.0123456789abcdef", "      --token-ttl duration                   The duration before the token is automatically deleted (e.g. 1s, 2m, 3h). If set to '0', the token will never expire (default 24h0m0s)", "", "Global Flags:", "      --log-file string   If non-empty, use this log file", "      --rootfs string     [EXPERIMENTAL] The path to the 'real' host root filesystem.", "      --skip-headers      If true, avoid header prefixes in the log messages", "  -v, --v Level           log level for V logs", "", "Use \"kubeadm init [command] --help\" for more information about a command.", "", "error: unknown flag: --skip-preflight-checks"], "stdout": "", "stdout_lines": []}
	to retry, use: --limit @/home/mrajanna/workspace/src/github.com/gluster/gluster-kubernetes/vagrant/site.retry

PLAY RECAP *********************************************************************
master                     : ok=28   changed=23   unreachable=0    failed=1   
node0                      : ok=23   changed=20   unreachable=0    failed=0   
node1                      : ok=23   changed=20   unreachable=0    failed=0   
node2                      : ok=23   changed=20   unreachable=0    failed=0   

==> master: An error occurred. The error will be shown after all tasks complete.
An error occurred while executing multiple actions in parallel.
Any errors that occurred are shown below.

An error occurred while executing the action on the 'master'
machine. Please handle this error then try again:

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

**Comment by nixpanic** [comment]
Please try [this commit](https://github.com/gluster/gluster-kubernetes/pull/545/commits/73aafb11912eda2804266d37895de18c99d1a706) which I added to PR#545 to get things working again.

**Comment by Madhu-1** [comment]
after removing `--skip-preflight-checks` it works

**Comment by Madhu-1** [comment]
after making the changes, heketi is not coming up

`
heketi topology loaded.
Saving /tmp/heketi-storage.json
secret/heketi-storage-secret created
endpoints/heketi-storage-endpoints created
service/heketi-storage-endpoints created
job.batch/heketi-storage-copy-job created
Error waiting for job 'heketi-storage-copy-job' to complete.
`

**Comment by nixpanic** [comment]
This should get fixed with #551
